rx-java/src/test/java/io/vertx/rx/java/test/JavaIntegrationTest.java

```diff
@@ -530,7 +530,6 @@ public void testHttpClient() {
     await();
   }
 
-  @Test
```

Comment:
I think for now it cannot pass until the HttpClientRequestBuilder is merged, so it will be continued in another branch with http-client-request-builder name